### DISCUSSION TITLE
Как в таблице циклов сделать разделение доступа?

### DISCUSSION THREAD

**Discussion by belikovmaxim** [totumonline/totum-ru-issues-and-discussions#81] [Как это сделать?] [answered]
Есть таблица циклов, есть префильтр, а так же пользователь которому в системную таблицу (Users) добавили поле Company.
Как сделать видимость наборов данных только тех, которые относятся к этому пользователю?
То есть Company = User.Company

**Comment by ttm-alexeypolunin** [comment, marked as answer]
### [Видео на YouTube —>](https://youtu.be/SWT1wmdlOus)

**[Учебный курс: Как настроить доступы к циклам для пользователей —>](https://docs.totum.online/how-to-configure-loop-accesses-for-users)**

Коды использованные в видео: 

**fl_company:**
```
= : if(condition: $#nr = 1; then: "*ALL*"; else: $sel)

sel: select(table: 'users'; field: 'company_s'; where: 'id' = $#nu)
```
**creator_id:**
```
= : set(table: $#ntn; field: $#nf + $sel[0]; where: 'id' = #id; log: true)

sel: select(table: 'users'; field: 'add_users'; where: 'id' = $#nfv)
```
